### ISSUE TITLE
Maybe.fromJust: Nothing exception

### ISSUE THREAD

**Issue by eskimor** [ghcjs/ghcjs#563]
If I use all.js I get the message:

```
Maybe.fromJust: Nothing 
```

in the browser console. It does not happen with the standard setup. (Actually it did happen before with standard out.js, but I managed to get rid of it by changing my code.) 

I am fully aware that this is not really much information, I am just hopping that maybe someone knowledgeable has some clue what could be the reason or help me track it down. 

([code](https://github.com/gonimo/gonimo))

Thank you!

**Comment by dmjio** [comment]
`ghcjs-dom-jsffi` uses `fromJust` a /lot/. If I had to guess, it would be in code where you're referencing that package. Have you tried printing the console until the error appears? This way you can rule out sections of code.

**Comment by eskimor** [comment]
Thank you @dmjio for this hint. Yeah, when it happend also with out.js I had just a few lines of code left where the cause could have been, but neither me nor the functions I referenced used fromJust - it had to be deeper down, thanks for the hint with ghcjs-dom-jsffi!

For now I managed to circumvent the error by not using all.js but instead cating rts.js lib.js and out.js together myself and sending this through the closure compiler with simple optimizations, this finally works with no runtime errors :-) yeah!
